worker/backend/backend.go

```diff
@@ -61,11 +65,32 @@ func (b *Backend) Capacity() (capacity garden.Capacity, err error) { return }
 
 // Create creates a new container.
 //
-// Errors:
-// * When the handle, if specified, is already taken.
-// * When one of the bind_mount paths does not exist.
-// * When resource allocations fail (subnet, user ID, etc).
-func (b *Backend) Create(spec garden.ContainerSpec) (container garden.Container, err error) {
+func (b *Backend) Create(gdnSpec garden.ContainerSpec) (container garden.Container, err error) {
+	var (
+		oci *specs.Spec
+		ctx = namespaces.WithNamespace(context.Background(), b.namespace)
+	)
+
+	oci, err = bespec.OciSpec(gdnSpec)
+	if err != nil {
+		err = fmt.Errorf("failed to convert garden spec to oci spec: %w", err)
+		return
+	}
+
+	cont, err := b.client.NewContainer(ctx,
+		gdnSpec.Handle, map[string]string(gdnSpec.Properties), oci,
```

Comment:
We can probably remove this conversion, Go should be able to implicitly convert the named type.